### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
This PR corrects misspellings identified by the [check-spelling action](https://github.com/marketplace/actions/check-spelling).

The misspellings have been reported at https://github.com/jsoref/helm-diff/commit/2c8c50b1828fdd4aabe38a493a186ae15c3e837a#commitcomment-46569570

The action reports that the changes in this PR would make it happy: https://github.com/jsoref/helm-diff/commit/0cd3196c53cf0732904c3ea60bdba5eee7cafc25

Note: this PR does not include the action. If you're interested in running a spell check on every PR and push, that can be offered separately.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,7 +46,7 @@ The Helm Diff Plugin
 
 * Shows a diff explaining what a helm rollback would change:
     This fetches the currently deployed version of a release
-  and compares it to adeployed versions of a release, that you 
+  and compares it to deployed versions of a release, that you
```
Comment (jsoref) [inline README.md:49]: initially, i was going to fix this to `a deployed`, but that didn't agree with `versions`. You could do:
```suggestion
  and compares it to a deployed version of a release, that you 
```
instead...

File: README.md [hunk 2]
```diff
@@ -184,7 +184,7 @@ Global Flags:
 ```
 $ helm diff rollback -h
 
-This command compares the laset manifests details of a named release 
+This command compares the latest manifest details of a named release
```
Comment (jsoref) [inline README.md:187]: It's possible that this wants a possessive, here I think we have a pair of adjectives

**Comment by jshuhnow** [comment]
Would any of the maintainers merge this please?

**Comment by jsoref** [comment]
@mumoshu?

File: README.md (hunk 1 above)
Comment (mumoshu) [reply README.md:49]: It's better, but still doesn't make sense? Perhaps we should instead say...

```suggestion
  and compares it to the previously deployed versions of the release, that you 
```

File: cmd/root.go [hunk 3]
```diff
@@ -20,7 +20,7 @@ The Helm Diff Plugin
 
 * Shows a diff explaining what a helm rollback would change:
     This fetches the currently deployed version of a release
-  and compares it to adeployed versions of a release, that you 
+  and compares it to deployed versions of a release, that you
```
Comment (mumoshu) [inline cmd/root.go:23]:
```suggestion
  and compares it to the previously deployed version of the release, that you 
```

**Comment by jsoref** [comment]
@mumoshu ?

**Comment by jsoref** [comment]
?

**Review by mumoshu — APPROVED** [review]
LGTM. Thanks for your contribution!

**Pull request merged**